src/components/indonesia/Dashboard.vue

```diff
@@ -140,6 +140,15 @@
                 treatment: '-'
             }
         },
+        filters: {
+            separator: function (value) {
+                if (!value) return ''
+                var formattedValue = ''
+                value = value.toString().split('').reverse().join('')
+                for(var i = 0; i < value.length; i++) if(i%3 == 0) formattedValue += value.substr(i,3)+'.'
```

Comment:
we already use `.` as default separator for decimal/float value. Ex: percent on the dashboard. It will be confusing if we use the same separator for a thousand.

Reply:
Oke use `,` instead